src/vmware/azext_vmware/_params.py

```diff
@@ -21,23 +21,23 @@ def load_arguments(self, _):
         c.argument('private_cloud', options_list=['--private-cloud', '-c'], help='Name of the private cloud.')
 
     with self.argument_context('vmware private-cloud') as c:
-        c.argument('circuit_primary_subnet', help='A /30 subnet for the primary circuit in the Express Route to configure routing between your network and Microsoft\'s Enterprise edge (MSEEs) routers.')
-        c.argument('circuit_secondary_subnet', help='A /30 subnet for the secondary circuit in the Express Route to configure routing between your network and Microsoft\'s Enterprise edge (MSEEs) routers.')
         c.argument('cluster_size', help='Number of hosts for the default management cluster. Minimum of 3 and maximum of 16.')
-        c.argument('network_block', help='A subnet at least of size /22. Make sure the CIDR format is conformed to (A.B.C.D/X) where A,B,C,D are between 0 and 255, and X is between 0 and 22.')
+        c.argument('internet', help='Connectivity to internet. Specify "Enabled" or "Disabled".')
 
     with self.argument_context('vmware cluster') as c:
         c.argument('name', options_list=['--name', '-n'], help='Name of the cluster.')
         c.argument('sku', help='The product SKU.')
         c.argument('size', help='Number of hosts for the cluster. Minimum of 3 and a maximum of 16.')
+        c.argument('hosts', nargs='+', help='A cluster\'s hosts in the private cloud.')
 
     with self.argument_context('vmware private-cloud create') as c:
         c.argument('name', options_list=['--name', '-n'], help='Name of the private cloud.')
         c.argument('sku', help='The product SKU.')
-        c.argument('internet', help='Connectivity to internet. Specify "Enabled" or "Disabled".')
         c.argument('vcenter_password', help='vCenter admin password.')
         c.argument('nsxt_password', help='NSX-T Manager password.')
         c.argument('accept_eula', help='Accept the end-user license agreement without prompting.')
+        c.argument('network_block', help='A subnet at least of size /22. Make sure the CIDR format is conformed to (A.B.C.D/X) where A,B,C,D are between 0 and 255, and X is between 0 and 22.')
+        c.argument('mi_system_assigned', help='Enables a system assigned identity.')

```

Comment:
Please use the first person voice
```suggestion
        c.argument('mi_system_assigned', help='Enable a system assigned identity.')

```